ui/pages/index.js

```diff
@@ -67,66 +68,48 @@ export default function Dashboard() {
                     </MetricsButton>
                   )}
                 </Grid>
+                </Grid>
               </PaperWithTitle>
             );
           }}
         </MetricsContainer>
-
-        <MesheryServerVersionContainer>
-          {({ serverVersion }) => (
-            <PaperWithTitle title="Release">
-              <Grid item xs={6}>
-                <Typography variant="subtitle1" sx={{ fontWeight: "bold" }}>
-                  Channel Subscription
-                </Typography>
-                {serverVersion.release_channel}
-              </Grid>
-              <Grid item xs={6}>
-                <Typography variant="subtitle1" sx={{ fontWeight: "bold" }}>
-                  Version
-                </Typography>
-                {getMesheryVersionText(serverVersion)}
-                <Link
-                  href={`https://docs.meshery.io/project/releases${
-                    serverVersion.release_channel === "edge" ? "" : "/" + serverVersion.build
-                  }`}
-                  target="_blank"
-                >
-                  <OpenInNewIcon sx={{ fontSize: theme.spacing(2) }} />
-                </Link>
-              </Grid>
-              <Grid item xs={12} sx={{ mt: theme.spacing(4) }}>
-                {serverVersion.outdated ? (
-                  <>
-                    <Typography variant="subtitle1" sx={{ fontWeight: "bold" }}>
-                      Newer version of Meshery available:
-                    </Typography>
-                    <Link
-                      href={`https://docs.meshery.io/project/releases${
-                        serverVersion.release_channel === "edge" ? "" : "/" + serverVersion.build
-                      }`}
-                      target="_blank"
-                    >
-                      {serverVersion.latest}
-                    </Link>
-                  </>
-                ) : (
-                  <Typography variant="body1" sx={{ fontStyle: "italic" }}>
-                    Running latest version of Meshery
-                  </Typography>
-                )}
-              </Grid>
-            </PaperWithTitle>
-          )}
-        </MesheryServerVersionContainer>
       </Stack>
     </PaperWithTitle>
   );
+  
+const ShowServiceMesh = () => {
+  return(
+    <div
+    style={{
+      margin: "auto",
+      display : "flex",
+      justifyContent : "center",
+      alignItems : "center",
+      flexDirection : "column",
+    }}
+  >
+    <Typography style={{ fontSize : "1.5rem", marginBottom : "2rem" }} align="center" color="textSecondary">
```

Comment:
@YashKamboj Could we avoid inline styles in refactoring? I know we are using [styled API](https://mui.com/system/styled/) so no` makestyles `will be used. We could probably use this `sx` prop. Reference - https://mui.com/system/the-sx-prop/#typography or use plain CSS separate sheet ref- https://mui.com/material-ui/migration/migration-v4/#fix-css-specificity-optional.

Let's define all the criteria to get the best out of this refactoring process.